### ISSUE TITLE
Stylize money input to make it more user appealing

### ISSUE THREAD

**Issue by Daveless** [EternoSeeker/Slot-Machine#7]
![image](https://github.com/EternoSeeker/Slot-Machine/assets/44807207/c1ffe45a-e713-4bfa-b7d5-91ebb2f5dc85)

![image](https://github.com/EternoSeeker/Slot-Machine/assets/44807207/06b02813-9e97-43f6-b0d1-3261ce36f11d)

**Comment by jpacsai** [comment]
Hi, I would like to work on this issue

**Comment by Itssshikhar** [comment]
Hi! Would like to take this issue

**Comment by Piyush07p** [comment]
can u assign it to me

**Comment by EternoSeeker** [comment]
Please also mention in brief how exactly you wish to solve it

**Comment by EternoSeeker** [comment]
Also you may make a Pull Request directly regarding this issue and we can discuss and review the changes

**Comment by Piyush07p** [comment]
hey @EternoSeeker  can you briefly elaborate what actually do u mean by this --> Stylize money input to make it more user appealing.
As of my understanding it seems like you don't want alert box to take money input but want a customized box to take input for amoney

**Comment by EternoSeeker** [comment]
yes a customized box would be good, both for alerts and input prompts
